distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc

```diff
@@ -48,15 +48,15 @@ On Windows, this means they must be under the same drive.
 
 A third party tool will have to be used to persist the logs in a highly available manner.
 
-* Edit the `${home_directory}/etc/org.ops4j.pax.logging.cfg` file to enable log4j2, following the steps in <<_enabling_fallback_audit_logging, Enabling Fallback Audit Logging>>.
+* Edit the `${home_directory}/etc/org.ops4j.pax.logging.cfg` file to enable log4j2, following the steps in <<{managing-prefix}enabling_fallback_audit_logging, Enabling Fallback Audit Logging>>.
 * Then put the appropriate log4j2 appender in `${home_directory}/etc/log4j2.config.xml` to send logs to the chosen third party tool.
 See https://logging.apache.org/log4j/2.x/manual/appenders.html[Log4j Appenders].
 
 ===== Shared Storage Provider
 
 The storage provider must be in a location that is shared between the two ${branding} nodes and must be highly available.
 If hardening the Highly Available Cluster, this shared storage provider must be trusted/secured.
-One way to accomplish this is to use the default <<org.codice.ddf.catalog.content.impl.FileSystemStorageProvider,Content File System Storage Provider>> and configure it to point to a highly available shared directory.
+One way to accomplish this is to use the default <<{application-prefix}org.codice.ddf.catalog.content.impl.FileSystemStorageProvider,Content File System Storage Provider>> and configure it to point to a highly available shared directory.
```

Comment:
❓ Why does this one require this prefix? Was this link not working before?

Reply:
It was working, but when I separated the different section into smaller documents, they _will_ break